src/renderer/main.js

```diff
@@ -302,6 +295,7 @@ function backToList () {
     var contentTag = document.querySelector('.content')
     if (contentTag) contentTag.scrollTop = 0
 
+    // TODO dcposch: is this still required with React?
```

Comment:
I'll try it and see if it's slower. Reason I haven't done that before to is to allow collapsing multiple `update()` calls into one.